File: src/Suframa.Sciex.UserInterface/www/app/view-model/EnderecoPessoaJuridicaVM.ts
import { manterEnderecoVM } from './ManterEnderecoVM';

export class enderecoPessoaJuridicaVM {
	endereco: manterEnderecoVM;
	idPessoaJuridica?: number;
	idCep: number;
	idUnidadeCadastradora?: number;
	numeroEndereco: string;
	complemento: string;
	pontoReferencia: string;
	telefone?: number;
	ramal?: number;
	email: string;
}

File: src/Suframa.Sciex.CrossCutting.DataTransferObject/ViewModel/EnderecoPessoaJuridicaVM.cs
namespace Suframa.Sciex.CrossCutting.DataTransferObject.ViewModel
{
    public class EnderecoPessoaJuridicaVM
    {
        public string Complemento { get; set; }

        public string Email { get; set; }

        //CEP
        public ManterEnderecoVM Endereco { get; set; }

        public int? IdCep { get; set; }

        // Pessoa Juridica
        public int? IdPessoaJuridica { get; set; }

        public int IdUnidadeCadastradora { get; set; }

        public string NumeroEndereco { get; set; }
        public string PontoReferencia { get; set; }

        public string Ramal { get; set; }
        public string Telefone { get; set; }
    }
}